heron/config/src/yaml/conf/yarn/heron_internals.yaml

```diff
@@ -65,6 +65,9 @@ heron.streammgr.cache.drain.size.mb: 100
 # For efficient acknowledgements
 heron.streammgr.xormgr.rotatingmap.nbuckets: 3
 
+# The max size of the memory pool for any type of message
+heron.streammgr.mempool.size.mb: 512
```

Comment:
Is this config for limiting the # of messages in the mempool or the total byte size in this mempool? This config is different than the other above 9 config.

Could you make them consistent and update the comment and even the config name to avoid confusion?